common-docs/beta.md

```diff
@@ -0,0 +1,27 @@
+# Beta editor
+
+New features, bug fixes, and other changes that aren't in the current release version of the editor are available to try in the Beta editor. Most MakeCode editors will let you preview some of what's coming in the next release by trying out the [Beta](@homeurl@/beta).
+
+## How can I try the Beta? #using-beta
+
+Trying the Beta version is simple. Go to the **Settings** menu (click on the ⚙️ symbol) and select **About**.
+
+![Settings menu](/static/beta/settings-about.jpg)
+
+Now, in the "About" window, click on the **Experiments** button.
+
+![Experiments button](/static/beta/about-experiments.jpg)
+
+In the "Experiments" window, find the **Beta** card and click on it. The Beta editor will now load.
+
+![Beta card](/static/beta/beta-card.jpg)
+
+Have fun with the Beta editor!
+
+### ~alert
+
```

Comment:
We probably need to add a caution note here about the possibility of any beta projects getting lost, becoming incompatible, and so on... Something like "Beta is for testing and experimentation **only**. We do not recommend using it for actual projects. It's under constant development, and any projects made in Beta may be lost or become nonfunctional in future updates."